src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -292,10 +290,9 @@ public sealed partial class RefreshPropertiesAttribute : System.Attribute
         public static readonly System.ComponentModel.RefreshPropertiesAttribute Default;
         public static readonly System.ComponentModel.RefreshPropertiesAttribute Repaint;
         public RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties refresh) { }
-        public System.ComponentModel.RefreshProperties RefreshProperties { get; }
-        public override bool Equals(object value) { throw null; }
+        public System.ComponentModel.RefreshProperties RefreshProperties { get { throw null; } }
+        public override bool Equals(object obj) { throw null; }
```

Comment:
This would be a breaking change - either update here or in the source - cc @maryamariyan, @safern

Reply:
Is it actually a breaking change?  I don't know if the compiler binds to the override's argument list or the original declaration (I assume the original declaration (except maybe in a base-invocation expression?))

Reply:
I am not sure myself. cc @jaredpar, @agocke can you help answer if renaming the parameter name on an overriden `Equals` method is considered a breaking change?